composer.lock

```diff
@@ -1176,37 +1176,37 @@
         },
         {
             "name": "piwik/ini",
-            "version": "1.0.6",
+            "version": "1.0.7",
             "source": {
                 "type": "git",
-                "url": "https://github.com/piwik/component-ini.git",
-                "reference": "bd2711ba4d5e20e4ca09b6829dc2831576b59dc3"
+                "url": "https://github.com/matomo-org/component-ini.git",
+                "reference": "6dec8e1f7f31ed6c3656eb997e771f23a1bdf2e1"
             },
             "dist": {
                 "type": "zip",
-                "url": "https://api.github.com/repos/piwik/component-ini/zipball/bd2711ba4d5e20e4ca09b6829dc2831576b59dc3",
-                "reference": "bd2711ba4d5e20e4ca09b6829dc2831576b59dc3",
+                "url": "https://api.github.com/repos/matomo-org/component-ini/zipball/6dec8e1f7f31ed6c3656eb997e771f23a1bdf2e1",
+                "reference": "6dec8e1f7f31ed6c3656eb997e771f23a1bdf2e1",
                 "shasum": ""
             },
             "require": {
                 "php": ">=5.3.3"
             },
             "require-dev": {
                 "athletic/athletic": "0.1.*",
-                "phpunit/phpunit": "~4.0"
+                "phpunit/phpunit": "^4.8.36"
             },
             "type": "library",
             "autoload": {
                 "psr-4": {
-                    "Piwik\\Ini\\": "src/"
+                    "Matomo\\Ini\\": "src/"
```

Comment:
That doesn't seem correct. Did you create the 1.x release from a branch where the namespace was already renamed?

Reply:
Must have forgotten to select the right branch... fixed it